charts/tempo-distributed/templates/distributor/service-distributor.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "tempo.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "tempo.distributorLabels" . | nindent 4 }}
+spec:
```

Comment:
Please make it possible to add annotations to this service (or maybe all services/deployments?)